.github/workflows/quic.yml

```diff
@@ -139,59 +139,16 @@ jobs:
       build: -Perf
       ref: ${{ github.event.client_payload.sha || github.event.client_payload.ref || inputs.ref || 'main' }}
 
-  #
-  # Manage Azure VMs, TODO: blocks if resources are not available. Waits for it to become available.
-  #
-  manage-azure-vms:
-    name: Manage Azure VMs
-    runs-on: windows-latest
-    outputs:
-      full-matrix: ${{ steps.generate-full-matrix.outputs.matrix }}
-      azure-matrix: ${{ steps.generate-azure-matrix.outputs.matrix }}
-    steps:
-      - name: Checkout
-        uses: actions/checkout@v2
-      - name: Install Latest Az PowerShell Module
-        shell: pwsh
-        run: |
-          Install-Module -Name Az -Force -AllowClobber -Repository PSGallery
-          Update-Module Az -Force
-          Get-Module 'Az' | where {([string]($_.Version)).StartsWith('9.3.0')} | Remove-Module
-          Get-Module -Name Az -ListAvailable
-      - name: Login to Azure
-        uses: azure/login@v1
-        with:
-          client-id: ${{ secrets.AZURE_CLIENT_ID }}
-          tenant-id: ${{ secrets.AZURE_TENANT_ID }}
-          subscription-id: ${{ secrets.AZURE_SUBSCRIPTION_ID }}
-          enable-AzPSSession: true
-      - name: Run manage-azure-vms.ps1
-        run: ./.github/workflows/manage-azure-vms.ps1 -GithubPatToken ${{ secrets.PERSONAL_ACCESS_TOKEN }}
-        shell: pwsh
-      - id: generate-full-matrix
-        run: echo "matrix=$(jq -c '.' ./.github/workflows/processed-matrix.json)" >> $GITHUB_OUTPUT
-        shell: bash
-      - id: generate-azure-matrix
-        run: echo "matrix=$(jq -c '.' ./.github/workflows/azure-matrix.json)" >> $GITHUB_OUTPUT
-        shell: bash
 
   #
-  # Create temporary VMs
+  # Manage and Create temporary VMs
   # NOTE: tag == env
   #
   create-azure-vms:
-    needs: [manage-azure-vms]
-    name: Create temporary Azure VMs
-    uses: microsoft/netperf/.github/workflows/create-azure-vms.yml@create-azure-machines
-    strategy:
-      fail-fast: false
-      matrix:
-        include: ${{fromJson(needs.manage-azure-vms.outputs.azure-matrix)}}
+    name: Manage and Create Temporary VMs
+    uses: microsoft/netperf/.github/workflows/create-azure-vms.yml@jackhe/robust-vms
```

Comment:
Why is this referencing `jackhe/robust-vms`?

Reply:
Will update to @main once we approve this PR, and get this merged to main. 

This is so that we can run the callable workflow in this PR's version.